ocs_ci/ocs/ocp.py

```diff
@@ -401,3 +401,64 @@ def switch_to_default_rook_cluster_project():
         bool: True on success, False otherwise
     """
     return switch_to_project(defaults.ROOK_CLUSTER_NAMESPACE)
+
+
+def get_default_storage_class():
+    """
+    Get the default StorageClass(es)
+
+    Returns:
+        list: StorageClass(es) list
+
+    """
+    count = 0
+    default_storage_class = []
+    default_sc_obj = OCP(kind='StorageClass')
+    default_sc_objs = default_sc_obj.get()['items']
+    for i in range(len(default_sc_objs)):
+        if 'annotations' in default_sc_objs[i]['metadata']:
+            ret = default_sc_objs[i]['metadata']['annotations'][
+                'storageclass.kubernetes.io/is-default-class'
+            ]
+            if ret == 'true':
+                count = count + 1
+                default_storage_class.append(
+                    default_sc_objs[i]['metadata']['name'])
+    if count == 1:
+        log.info("One default StorageClass found")
+        return default_storage_class[0]
+    elif count > 1:
+        log.info("More than one default StorageClass found")
+        return default_storage_class
+    else:
+        log.info("No default StorageClass found")
+
+
+def change_default_storageclass(scname):
+    """
+    Change the default StorageClass to the given SC name
+
+    Args:
+        scname (str): StorageClass name
+
+    Returns:
+        bool: True on success, False otherwise
```

Comment:
not accurate - you are not returning False anywhere but an exception will be raised.
Not a big deal for now

Reply:
done